src/jbi/whiteboard_actions/README.md

```diff
@@ -0,0 +1,20 @@
+## Looking to add a new action?
+"actions" are python functions that, once added into `src/jbi/whiteboard_actions/` dir,
+can be run by updating the `action` attribute of the yaml config file.
+
+### Create a new action...
+Let's create a `new_action`!
+1. First, add a new python file called "my_team_action" in the `src/jbi/whiteboard_actions/` dir.
+1. Add the python function `init` to a new "my_team_action" module, let's use the following:
+    ```python
+    def init(whiteboard_tag, jira_project_key, optional_param=42):
+        return lambda payload, context: print("Hi world!")
+    ```
+    1. In the above example whiteboard_tag and jira_project_key are required
+    1. optional_param, which has a default value--is not required to run this action
+    1. init returns a `__call__`able object that the system calls with the bugzilla request and yaml configuration objects
+1. Using `payload` and `context` perform the desired processing!
+    1. `payload` is bugzilla webhook payload
+    1. `context` is the full yaml blob as a python dict (if additional context is desired)
```

Comment:
If I undestand correctly, here we would have additional fields like `description` and `contact`. I believe that if an action requires more input values, it could rely on parameters instead. And this makes this `context` a bit useless I think

Reply:
https://github.com/mozilla/jira-bugzilla-integration/pull/14/commits/e1ba4cb152c9eb92ce373727c1331cfa59a0c886